buoi9/main.js

```diff
@@ -0,0 +1,93 @@
+//bai 1:nhap luong nhan vien,tinh thue thu nhap va luong rong
+const getsalary =(salary) =>{
```

Comment:
```suggestion
const getSalary = salary =>{
```

Đọc lại quy tắc đặt tên hàm.
Nếu hàm chỉ có duy nhất 1 param thì có thể bỏ dấu `()`